automatic/nomacs/update.ps1

```diff
@@ -21,8 +21,8 @@ function global:au_BeforeUpdate  { Get-RemoteFiles -NoSuffix -Purge }
 
 function global:au_GetLatest {
     $download_page = Invoke-WebRequest -Uri $releases -UseBasicParsing
-    $url     = $download_page.links | ? href -like '*/nomacs-*' | % href | select -First 1
-    $version = $url -split '-|\.zip' | select -Last 1 -Skip 1
+    $url     = $download_page.links | ? href -like '*/nomacs-*x64.msi' | % href | select -First 1
+    $version = $url -split '\/' | select -Last 1 -Skip 1
     @{
         Version      = $version
         URL64        = 'http://download.nomacs.org/nomacs-setup-x64.msi'
```

Comment:
Please change URL for install version to be taken from the github releases.

Reply:
fixed, but had to rip out the 32bit url, as it's not present on github releases

Reply:
That is fine, as I don't think official 32bit editions are even provided by the developers anymore.

However, removing the 32bit URL from `update.ps1` that also means it will need to be removed from the install script (the file argument), same with references to url32 in `VERIFICATION.txt`, could you please do that?
Would be great if you could also add the same 32bit check in the `nomacs` package that is used in the `.portable` package.

Reply:
DOH! thank's for catching this! - see GH-1494